pkg/app/app.go

```diff
@@ -197,11 +191,17 @@ func CreateApp() App {
 		}
 		app.log.Info("Kubernetes version: %v", kubeServerInfo)
 	case kapierrors.IsNotFound(err) || kapierrors.IsUnauthorized(err) || kapierrors.IsForbidden(err):
+		app.log.Error("the server could not find the requested resource")
+		app.log.Error(err.Error())
+		os.Exit(1)
 	default:
 		app.log.Error(err.Error())
 		os.Exit(1)
 	}
 
+	app.log.Debug("Connecting Dao")
+	app.dao, err = dao.NewDao(app.config.Dao, app.log.Logger)
```

Comment:
Should we handle this error or should we remove the error?

Reply:
We should be handling it in case the etcd config file is not found.